controller/auth.js

```diff
@@ -4,11 +4,17 @@ require('dotenv').config({ slient: true });
 
 const db = require("../models/index.js");
 const models = db.sequelize.models;
-const { statusCode } = require('../utils')
+const { statusCode } = require('../utils');
 
 async function login (req, res) {
   if (req.user)
-    return res.status(statusCode.BAD_REQUEST).send("Already signed in");

```

Comment:
Login validation em còn phải add validation ở các function khác không phải log in() để check coi đã log in hay chưa :D

Reply:
Hiện tại em đang dùng middleware có sẵn của jwt để lấy token á, nhưng em chỉ áp dụng cho route này, vậy thì em sửa trong file index để thêm middleware cho mọi route luôn đúng không ạ?

Reply:
ah không hình như mình đang misunderstand. 
Login validation nghĩa là 
khi em get user, e phải validate coi nó đã login hay chưa. Nếu nó chưa login (means nó ko có login token gửi về) thì không cho nó get user.